cloudscript/cloudscript/CreateImage.ps1

```diff
@@ -0,0 +1,93 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Image name to be created
+    [string]$imageName,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Image resource group for above image
+    [string]$imageResourceGroup,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Blob sas uri from which image to be created
+    [string]$blobUri,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateSet('Windows','Linux')]
+    # os type for image: Linux/Windows
+    [string]$osType,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Resource group containing azure device
+    [string]$saasResourceGroup,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Azure ASE device name
+    [string]$deviceName,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Azure subscription id
+    [string]$subscriptionId,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # Access key for azure subscription access
```

Comment:
This is the access key required to access the storage account which houses the vhd. We should probably move this step to the top just after the step where we ask the customer to provide the blobURI

Reply:
this is for accessing the subscription and not the storage account